VMAccess/errorclarification.py

```diff
@@ -0,0 +1,10 @@
+import enum
+
+class ErrorCode(enum.Enum):
+    REPAIR_DISK_ERROR = -2
+    CHECK_DISK_ERROR = -1
+
+    SYSTEM_ERROR = 0
+    
+    USER_PASSWORD_ERROR = 1
+    CHECK_REPAIR_DISK_ERROR = 4
```

Comment:
The naming here is confusing given that we have the system errors REPAIR_DISK_ERROR and CHECK_DISK_ERROR above.